.github/workflows/images-branch.yml

```diff
@@ -95,6 +97,12 @@ jobs:
       - name: Prepare tag (forks)
         run: |
           gh pr checkout ${{ inputs.pr }}
+
+          # Ensure the commit we've checked out matches our expected SHA from when we checked the PR's approval status above.
+          # This is a security measure to prevent any unreviewed commits from getting pulled into this action workflow.
+          # The format is "APPROVED:OPEN:<PR_COMMIT_SHA>", so compare the end of the string to the current commit.
+          [[ ${{needs.approval.outputs.decision}} =~ $(git rev-parse HEAD)$ ]]
```

Comment:
💅 This is my _preference_, not a requirement.

I'd prefer we use two outputs instead of matching against a long string multiple times. I find exact matches quite a bit easier to reason about.

Reply:
I hear you... I actually wrote it that way originally--I also prefer explicit over implicit, but it got kinda verbose, so I decided it was more readable/simpler this route.